Automattic/wp-calypso
# Hosting Overview: Use real storage data

Related to https://github.com/Automattic/dotcom-forge/issues/6384

## Proposed Changes

* Replace the dummy data used on `PlanCard` with the actual site storage, provided by the `PlanStorage` component.
* Since the `PlanStorageBar` component used on `PlanStorage` didn't quite fit the design needs, I've refactored it so it now accepts a `StorageBarComponent` prop.

## Testing Instructions

- Open the live preview
- Go to `/hosting-overview`
- Choose a site
- Check that the plan card shows the storage you are actually using, you can compare it with the data on `/wp-admin/media-new.php`

## Pre-merge Checklist
